streams/src/main/java/org/apache/kafka/streams/processor/internals/StateManagerUtil.java

```diff
@@ -83,8 +84,8 @@ static void wipeStateStores(final Logger log, final ProcessorStateManager stateM
         try {
             Utils.delete(stateMgr.baseDir());
         } catch (final IOException fatalException) {
-            // since it is only called under dirty close, we always swallow the exception
```

Comment:
Why do we now always wipe the state stores (even on clean close)? cc/ @guozhangwang

Reply:
We do not wipe state stores on clean close, but only in dirty close.

Reply:
`wipeStateStores` is only triggered if it is dirty close, under EOS.

Reply:
Ah I wasn't looking at the latest `trunk`, I see you fixed that in a recent PR. In that case we should leave the comment?